src/openrct2/network/NetworkServerAdvertiser.cpp

```diff
@@ -241,6 +242,11 @@ class NetworkServerAdvertiser final : public INetworkServerAdvertiser
         static constexpr char hexChars[] = {
             '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f',
         };
+
+        // Seed with non-deterministic random number.
+        std::random_device rd;
+        util_srand(rd());
+
```

Comment:
I think it would be better to just not use `util_rand` in this method at all and instead use the c++ random API directly.

Reply:
Done, also uses now uniform_int_distribution instead of modulo.